src/compiler/c.rs

```diff
@@ -77,6 +77,19 @@ impl ParsedArguments {
     }
 }
 
+pub fn normalized_extension(file: &Path) -> Option<&'static str> {
```

Comment:
I had actually mentioned this on a different PR a while ago, but it might be nicer to have an enum of C dialects to use here.